src/setup/sign-up-to-mergin-maps/index.md

```diff
@@ -1,67 +1,79 @@
 # How to Sign Up to Mergin Maps
 [[toc]]
 
-To start using the <MainPlatformNameLink /> service, you need to sign up. You can sign up either through <MainDomainNameLink /> or from your mobile device using the <MobileAppName />.
-
-When you register with <MainPlatformNameLink />, you will be automatically signed up as a free tier user with 100 MB of storage. To upgrade your account, see our <MainDomainNameLink id="pricing" desc="Subscription plans" />.
+To make full use of the <MainPlatformNameLink /> service, you need to sign up. You can sign up either through <MainDomainNameLink /> or from your mobile device using the <MobileAppName />.
 
 ::: tip
-One can view and download public projects even without an account, but creation of projects and synchronisation of geo-data only works when you are logged in.
+Public projects can be accessed and downloaded even without a <MainPlatformNameLink /> account, but projects can be managed, shared and synchronised only when you are logged in.
 :::
 
 ## From web
 
 1. Navigate to <MainDomainNameLink /> 
 2. Click on **Start for free**
-   ![mergin maps register](./mergin-web-register.jpg)
+   ![Mergin Maps start for free](./mergin-web-register.jpg "Mergin Maps start for free")
 
 3. Click on **Create your free account**
-   ![web create account](./mergin-web-create-account.png)
+   ![Mergin Maps create free account](./mergin-web-create-account.jpg "Mergin Maps create free account")
    
 4. Create your account by filling up the form and click on **Sign Up**. You will receive a confirmation email with a link to verify your email.
-   ![sign up](./mergin-web-sign-up.png)
+   ![Mergin Maps sign up form](./mergin-web-sign-up.jpg "Mergin Maps sign up form")
 
    ::: warning
    Check your spam folder if the confirmation email does not appear in your inbox after 5 minutes.
    :::
 
+5. Now your account needs a [workspace](../../manage/workspaces/)!
+- If you were invited to an existing workspace, you can simply **join** this workspace without creating a new one
+![Join workspace in Mergin Maps](./join-workspace.jpg "Join workspace in Mergin Maps")
+
+- If you want to create your own workspace, choose an appropriate name and click on **Create workspace**
+   ![Create workspace in Mergin Maps](./create-workspace.jpg "Create workspace in Mergin Maps")
+
+   :::tip
+   Projects, collaborations and subscriptions in <MainPlatformNameLink /> are tied to [workspaces](../../manage/workspaces/). If you create your own workspace, you will be automatically signed up as a free tier user with 100 MB of storage. To upgrade your subscription, see our <MainDomainNameLink id="pricing" desc="Subscription plans" />.
```

Comment:
```suggestion
   Projects, collaborations and subscriptions in <MainPlatformNameLink /> are tied to [workspaces](../../manage/workspaces/). If you create your own workspace, it will be automatically signed up as a free tier workspace with 100 MB of storage. To upgrade your subscription, see our <MainDomainNameLink id="pricing" desc="Subscription plans" />.
```

Reply:
Let's talk about workspaces instead of users :)